src/test/scala/latis/input/HapiReaderSpec.scala

```diff
@@ -45,7 +46,7 @@ class HapiReaderSpec extends FlatSpec with Matchers {
   it should "support scalar timeseries datasets" in {
     val parameters: List[Parameter] = List(
       ScalarParameter("time", "isotime", "UTC", Option(24), None),
-      ScalarParameter("x", "type", "units", None, None)
```

Comment:
All these tests used to use `"type"` as the Scalar type, but that's no longer a valid Scalar type and causes errors. So I decided to replace `"type"` with `"string"` everywhere it was used. The decision was pretty arbitrary.